modules/telestion-application/src/main/java/de/wuespace/telestion/application/loader/GenericLoaderConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package de.wuespace.telestion.application.loader;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * Use this {@link LoaderConfiguration} if you don't use any strictly typed Loader configuration.
+ *
+ * @author Ludwig Richter (@fussel178)
+ * @see LoaderConfiguration
+ * @see Loader
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record GenericLoaderConfiguration() implements LoaderConfiguration {
+}
```

Comment:
Replace with `UntypedLoaderConfiguration` :wink:

Reply:
That 😂

Reply:
Done in [`8cf7509` (#502)](https://github.com/wuespace/telestion-core/pull/502/commits/8cf750925efc880e4438c6f542f64be677842bf5)